values.yaml

```diff
@@ -1,33 +1,43 @@
-octopus:
-  statefulSet:
-    annotations: {}
-    labels: {}
-  pods:
-    annotations: {}
-    labels: {}
+statefulSetLabel:
+  environment: prod
+
+podLabels:
+  environment: prod
+
   # The Octopus server image. Visit https://hub.docker.com/r/octopusdeploy/octopusdeploy for the available versions.
   image: octopusdeploy/octopusdeploy:2020.3.1-linux
-  # Set this to 1 if you do not have a HA license.
-  replicaCount: 1
+
+  # Set this to 1 if you do not have a HA license. If you want to run Octopus Deploy with HA and have an HA license, change the replica count
+  replicaCount: 2
+
   # The Octopus admin username
-  username: admin
+  ADMIN_USERNAME: admin
```

Comment:
The different styles here of uppercase and lowercase is a little confusing. We should stick to one style for these values.

Reply:
I've originally been going with what the environment variables are. So, for example, if the environment variable is `ADMIN_USERNAME`, I've been using `ADMIN_USERNAME` for the value. That way, it looks more flush. Is that okay with you?